packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,77 +4,27 @@ This log was last generated on Mon, 26 Feb 2024 16:19:15 GMT and should not be m
+
 ### Bug Fixes
-- update styling to include space for back button ([PR #4079](https://github.com/azure/communication-ui-library/pull/4079) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Add missing permissions to storybook deploy action ([PR #4090](https://github.com/azure/communication-ui-library/pull/4090) by 73612854+palatter@users.noreply.github.com)
+
+- Update styling to include space for back button ([PR #4079](https://github.com/azure/communication-ui-library/pull/4079) by 94866715+dmceachernmsft@users.noreply.github.com)
 - Defect fixing the src of the inline image ([PR #4104](https://github.com/azure/communication-ui-library/pull/4104) by 9044372+JoshuaLai@users.noreply.github.com)
-- Fix for test coverage upload job in CI workflow ([PR #4118](https://github.com/azure/communication-ui-library/pull/4118) by 98852890+vhuseinova-msft@users.noreply.github.com)
-- Resolve issue where envHelper would return early on error throw when appsettings.json doesn't exist ([PR #4161](https://github.com/azure/communication-ui-library/pull/4161) by 73612854+palatter@users.noreply.github.com)
-- Revert "[Chat] Wrap ImageOverlay with theme provider and change some of the APIs (#4117)" ([PR #4131](https://github.com/azure/communication-ui-library/pull/4131) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
-- Fix hanging express.close if have unclosed connection ([PR #4138](https://github.com/azure/communication-ui-library/pull/4138) by 77021369+jimchou-dev@users.noreply.github.com)
 - Fix for an issue when head and body tags were added to messages with html type ([PR #4106](https://github.com/azure/communication-ui-library/pull/4106) by 98852890+vhuseinova-msft@users.noreply.github.com)
 - Fixing defect when a img is shared via drag and drop and there is no message content ([PR #4084](https://github.com/azure/communication-ui-library/pull/4084) by 9044372+JoshuaLai@users.noreply.github.com)
 - Fix color loss of raised hand icon on iOS ([PR #4089](https://github.com/azure/communication-ui-library/pull/4089) by 79475487+mgamis-msft@users.noreply.github.com)
 - Resolve issue where images previews would not sure in a message if the message also had a file attachment. ([PR #4167](https://github.com/azure/communication-ui-library/pull/4167) by 73612854+palatter@users.noreply.github.com)
-- update gallery logic to not include local participant in grid calculations. ([PR #4136](https://github.com/azure/communication-ui-library/pull/4136) by dmceachern@microsoft.com)
+- Update gallery logic to not include local participant in grid calculations. ([PR #4136](https://github.com/azure/communication-ui-library/pull/4136) by dmceachern@microsoft.com)
 - Change inline image cursor to pointer ([PR #4164](https://github.com/azure/communication-ui-library/pull/4164) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
```

Comment:
We could remove this one because this is a fix for a newly introduced bug during refactoring in this release